packages/vuetify/src/components/VOverlay/VOverlay.tsx

```diff
@@ -254,6 +256,21 @@ export const VOverlay = genericComponent<OverlaySlots>()({
       emit('afterLeave')
     }
 
+    const mergedStyle = computed(() => {
```

Comment:
Wouldn't it be easier to just change the style ordering?

![image](https://github.com/vuetifyjs/vuetify/assets/9064066/4e0e324d-6e02-4432-bc2d-549987f63be8)

Reply:
If only the order is changed, when `min-width` is not set in `v-menu`, `min-width: undefined` in `dimensionStyles` will override the default `min-width` in `contentStyles`, so I fixed it this way.

Reply:
Your proposed fix just seems to be overly complex on the surface. Surely there is a less engineered way to approach this. I'll think on it.